bot/controllers/channels.js

```diff
@@ -0,0 +1,85 @@
+module.exports = () => {
+  const util = require('apex-util');
+
+  const _run = (message) => {
+    const ctrls = [
+      {
+        cmd: '!chnl',
+        example: '!chnl <channel_name>,<channel_name>,<message>',
+        title: 'Msg Multiple Specific Channels',
+        desc: 'Msg multiple specific channels. Channels are case-sensitive.',
+        showWithHelp: true,
+        posTargetUser: null,
+        posSecondaryCmd: null,
+        regexSplitCharacter: null,
+        allowInDM: false,
+        resType: 'reply',
+        action: (message, ctrl, msg) => {
+          const chnls = msg.parsed[1].split(',');
+          util.log('Multiple Channels Parsing', chnls, 4);
+
+          chnls.map((chnl) => {
+            const targetChnl = message.guild.channels.find('name', chnl);
+            util.log('Asking API for Channel', targetChnl, 4);
+
+            if (targetChnl === null) {
+              return '"' + chnl + '" is not a known channel. Check text channels on the left to see what channels are available (They are case-sensitive)';
+            }
+            const msgs = msg.parsed;
```

Comment:
This prints out all the arguments passed into the string, including the command and selected channels. I suggest iterating over each of the arguments you really need. e.g: msg.parsed[2] and going forward and concatenate them with a space.